package/boot/arm-trusted-firmware-rockchip/Makefile

```diff
@@ -0,0 +1,51 @@
+#
+# Copyright (C) 2020 Tobias Maedel <openwrt@tbspace.de>
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=arm-trusted-firmware-rockchip
+PKG_RELEASE:=1
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL=https://github.com/ARM-software/arm-trusted-firmware
+PKG_SOURCE_DATE:=2020-01-03
+PKG_SOURCE_VERSION:=22d12c4148c373932a7a81e5d1c59a767e143ac2
+PKG_MIRROR_HASH:=64e249ef41cf3b1e468eaefdf1435498a9346af99cc0fd91bccd95d6af2e71a5
+
+PKG_LICENSE:=BSD-3-Clause
+PKG_LICENSE_FILES:=license.md
+
+PKG_MAINTAINER:=Tobias Maedel <openwrt@tbspace.de>
+
+include $(INCLUDE_DIR)/package.mk
+
+
+define Package/arm-trusted-firmware-rockchip
+    SECTION:=boot
+    CATEGORY:=Boot Loaders
+    TITLE:=ARM Trusted Firmware for Rockchip
+    DEPENDS:=@TARGET_rockchip_rk3399
+endef
+
+export GCC_HONOUR_COPTS=s
```

Comment:
A nicer way would be to set and/or strip the optimization flag?

Reply:
I'm not sure if I can influence this in the external build system of the ATF. 
OpenWRT does this in the include for u-boot as well: 
https://github.com/openwrt/openwrt/blob/master/include/u-boot.mk

arm-trusted-firmware-sunxi does so as well: 
https://github.com/openwrt/openwrt/blob/master/package/boot/arm-trusted-firmware-sunxi/Makefile#L34

If you know a nicer way on how to fix this, please let me know.

Reply:
This looks like Linux only solution, see our remarks in https://github.com/openwrt/openwrt/pull/2521

Reply:
This looks like Linux only solution, see our remarks in https://github.com/openwrt/openwrt/pull/2521

Reply:
See https://github.com/openwrt/openwrt/pull/2917#issuecomment-614329227